packages/lit-dev-content/site/tutorials/content/first-component/04.md

```diff
@@ -0,0 +1,104 @@
+By the end of this step, the shown word will automatically change every second.
+
+This can be achieved by using
+[`setInterval`](https://developer.mozilla.org/en-US/docs/Web/API/setInterval), a
+browser function which repeatedly calls a function with a fixed delay between
+each call.
+
+Define a method `tickToNextWord` that will increment `this.idx`.
+
+{% switchable-sample %}
+
+```ts
+// word-viewer.ts
+
+  render() {
+    ...
+  }
+
+  tickToNextWord = () => { this.idx += 1; };
+
+```
+
+```js
+// word-viewer.js
+
+  render() {
+    ...
+  }
+
+  tickToNextWord = () => { this.idx += 1; };
+
+```
+
+{% endswitchable-sample %}
+
+Calling this method increments our reactive internal state `idx` which will
+schedule an update.
```

Comment:
Is it worth mentioning why this is an arrow function to bind the context of `this`?

Reply:
Added an info aside! Great callout!